### PR TITLE
dep-check fix: extension must not depend on 3p

### PR DESCRIPTION
**Pull request by zhouyx**

### CODE REVIEW

**Comment by cramforce** [comment]
LGTM

File: build-system/dep-check-config.js [hunk 1]
```diff
@@ -116,6 +116,10 @@ exports.rules = [
   },
   {
     filesMatching: 'extensions/**/*.js',
+    mustNotDependOn: '3p/**/*.js',
+  },
+  {
+    filesMatching: 'extensions/**/*.js',
```
Comment (erwinmombay) [inline build-system/dep-check-config.js]: can we move this to the config that already exists? just do:

```js
  {
    filesMatching: '{src,extensions}/**/*.js',
    mustNotDependOn: '3p/**/*.js',
  },

```

Comment (zhouyx) [inline build-system/dep-check-config.js]: fixed

**Comment by erwinmombay** [comment]
LGTM

**Pull request merged**